testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/forms/BrowserFlowTest.java

```diff
@@ -219,6 +229,59 @@ public void testSwitchExecutionNotAllowedWithRequiredPasswordAndAlternativeOTP()
     }
 
 
+    @Test
+    public void testSocialProvidersPresentOnLoginUsernameOnlyPageIfConfigured() {
+        String testRealm = "test";
+        // Test setup - Configure the testing Keycloak instance with UsernameForm & PasswordForm (both REQUIRED) and OTPFormAuthenticator (ALTERNATIVE)
+        testingClient.server(testRealm).run(configureBrowserFlowWithRequiredPasswordFormAndAlternativeOTP("browser - copy 1"));
+
+        List<String> identityProviderAliases = new ArrayList<String>();
+        try {
+            SocialLoginTest socialLoginTest = new SocialLoginTest();
+            List<SocialLoginTest.Provider> socialProviders = Arrays.asList(GITHUB, GITLAB, GOOGLE);
+            try {
+                Method buildProviderMethod = socialLoginTest.getClass().getDeclaredMethod("buildIdp", new Class[]{SocialLoginTest.Provider.class});
```

Comment:
How about changing that method on class SocialLoginTest to be public? That could simplify this test quite a lot as there won't be reflection needed and it will remove some fragility (for example if the private method on SocialLoginTest is later renamed).

Reply:
@mposolda Right, that would work too. Will try it (reused the existing `SocialLoginTest` function, so I wouldn't need to define a new copy with the same / similar functionality).